Sources/ForageSDK/Foundation/Protocol/ForageUI.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  File.swift
+//  
+//
+//  Created by Danny Leiser on 7/27/23.
+//
+
+import UIKit
+
+public protocol ForageUI: UIView {
+    var placeholder: String? { get set }
+    var collector: VaultCollector { get set }
+    var textColor: UIColor? { get set }
+    var tfTintColor: UIColor? { get set }
+    var isSecureTextEntry: Bool { get set }
+    var textAlignment: NSTextAlignment { get set }
+    var font: UIFont? { get set }
```

Comment:
do we risk this affecting PIN concealment if I try to change this for the PIN elements?

Reply:
Good question I don't believe so, but I think I'm going to remove it for now anyways.